src/nvim/move.c

```diff
@@ -608,16 +611,30 @@ void changed_line_abv_curs_win(win_T *wp)
                    |VALID_CHEIGHT|VALID_TOPLINE);
 }
 
-// Make sure the value of wp->w_botline is valid.
-void validate_botline(win_T *wp)
+/// Make sure the value of curwin->w_botline is valid.
+void validate_botline(void)
+{
+  validate_botline_win(curwin);
+}
```

Comment:
We don't want these alias functions back. caller should spell out `curwin` if that is what is intended.